lib/modules/dosomething/dosomething_reportback/dosomething_reportback.forms.inc

```diff
@@ -37,6 +37,15 @@ function dosomething_reportback_form($form, &$form_state, $entity = NULL) {
       '#required' => TRUE,
       '#default_value' => $entity->nid,
     );
+    $form['camapgin_run_nid'] = array(
+      '#type' => 'entity_autocomplete',
+      '#title' => t('Campaign Run ID'),
+      '#description' => t("The campaign run node this reportback was submitted for."),
+      '#entity_type' => 'node',
+      '#bundles' => array('campaign_run'),
+      '#required' => TRUE,
+      '#default_value' => $entity->nid,
```

Comment:
@angaither ah yes - it does work then in that case! If there's an error on the form, it saves the field to `Comeback Clothes 2014 [id:1227]` so you don't have to fill this out again.